src/test/test-hashmap-plain.c

```diff
@@ -646,7 +646,7 @@ TEST(hashmap_get) {
 }
 
 TEST(hashmap_get2) {
-        _cleanup_(hashmap_free_free_freep) Hashmap *m = NULL;
+        _cleanup_hashmap_free_free_free_ Hashmap *m = NULL;
```

Comment:
This sounds like a candidate to be dropped? It's hardly used and we aim to port remaining use of such _free_free funcs to proper hashmap destructor?

Reply:
yeah, that one is a wild ;-)

Reply:
OK. I will set specific hash ops and drop _free_free_free and _free_free. But, I'd like to do that consistently, not only here, and not only for hashmap_free_free_free(). Hence, merge this PR as is now, and will open a follow-up PR.

Reply:
-> #36107.